app/admin/users.rb

```diff
@@ -90,6 +94,26 @@ def update
       redirect_to admin_user_path(resource)
     end
 
+    def upload_csv
+
+    end
+
+    def import_data
+      @users = []
+      @attributes = ["id", "name", "email", "slack_username", "slack_userid", "created_at", "updated_at", "optin",  "is_approved", "airtable_id", "tech_skill_names", "non_tech_skill_names"]
```

Comment:
curious - why restrict the attribute names? I see for projects you just grab the list from the column_names. Mainly worried that if we change the user table some day and forget to update this list there will be problems.

Reply:
yes this is an issue that we'll have to bring up with stephen - basically, the progbot volunteer table we're testing with is very different from the actual members table that i assume will be used in production. the actual members table has all these fields that have no reflection in the database. also, our table in progbot has all these fields that come from the volunteer form but aren't reflected in airtable. so basically, there are a lot of incompatibilities between the user model in airtable and the user model in progbot - and the fields i restricted in the attributes are the only ones that appear in our 'progbot' test users table - so i think we'll have to discuss what to do about all these user related issues.